python/desc/imsim/atmPSF.py

```diff
@@ -101,7 +104,8 @@ def __init__(self, airmass, rawSeeing, band, rng,
         self.screen_scale = screen_scale
         self.logger = logger
 
-        self.atm = galsim.Atmosphere(**self._getAtmKwargs())
+        ctx = multiprocessing.get_context('fork')
```

Comment:
I *think* 'spawn' here is safer than 'fork'.  See this [blog post](https://codewithoutrules.com/2018/09/04/python-multiprocessing/), for example.  That said, I don't have any actual experience with problems using either launch context, so if this works in tests, it's probably fine.

Reply:
It does sound safer based on that blog post, but when I used `spawn`, the python environment was getting imported from scratch multiple times--in fact many more times than just the number of processes specified.  This was slowing down the test code past my limited patience and was also outputting a lot of DM info messages.  However, when I increased the number of processes to 16, the calculation died in a way I didn't understand, so maybe `fork` is at issue here.

Reply:
fwiw, I did a run last night to check that the full RandomKnots instance catalogs would work for CCDs that succeeded and failed in Antonio's Theta testing, and there was no problem using nproc=16.  But since there are only 6 screens being computed, I assume this is unnecessary?  @jmeyers314 How many processes would you recommend for the `self.atm.instantiate(...)` call?

Reply:
>the python environment was getting imported from scratch multiple times--in fact many more times than just the number of processes specified.

I see, so maybe safer but slower.  It's certainly weird that the imports are more frequent than the number of processes though.

>How many processes would you recommend for the self.atm.instantiate(...) call?

There's no advantage to using more than the number of screens that I can see.

Reply:
I updated the code to set the number of processes to be the number of atmospheric phase screens if `nproc` is not otherwise set.

> It's certainly weird that the imports are more numerous than the number of processes though.

I was mistaken about this.  I didn't realize `test_atmPSF.py` was creating 100 different `AtmosphericPSF` objects.